src/dune_pkg/solver_env.mli

```diff
@@ -8,19 +8,61 @@ module Flag : sig
 
   val to_string : t -> string
 
+  val of_string_opt : string -> t option
+
   module Set : sig
     type flag := t
 
     type t
 
     val fold : t -> init:'a -> f:(flag -> 'a -> 'a) -> 'a
+
+    val mem : t -> flag -> bool
   end
 end
 
-type t = { flags : Flag.Set.t }
+module Sys_var : sig
+  type t =
```

Comment:
Some light docs for this module and the submodule would be appreciated

Reply:
done